solace-message-router/nestedtemplates/workspace-enabled-shared-resources.json

```diff
@@ -0,0 +1,122 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "workspaceName": {
+      "type": "string",
+      "metadata": {
+        "description": "Unique OMS Workspace Name for Log Analytics."
+      }
+    },
+    "workspaceRegion": {
+      "type": "string",
+      "defaultValue": "East US",
+      "metadata": {
+        "description": "OMS Workspace Region for Log Analytics."
+      },
+      "allowedValues": [
+        "East US",
+        "West Europe",
+        "Southeast Asia",
+        "Australia Southeast",
+        "Japan East",
+        "UK South",
+        "Central India",
+        "Canada Central"
+      ]
+    }
+  },
+  "variables": {
+    "workspaceTier": "Free",
+    "containersMarketplaceName": "Containers",
+    "syslogFacilities": [ "kern", "user", "daemon", "auth", "syslog", "uucp", "authpriv", "ftp", "cron", "local0", "local1", "local2", "local3", "local4", "local5", "local6", "local7" ]
+  },
+  "outputs": {
+    "workspaceId": {
+      "type": "string",
+      "value": "[reference(resourceId('Microsoft.OperationalInsights/workspaces/', parameters('workspaceName')), '2015-03-20').customerId]"
+    },
+    "workspaceKey": {
+      "type": "string",
+      "value": "[listKeys(resourceId('Microsoft.OperationalInsights/workspaces/', parameters('workspaceName')), '2015-03-20').primarySharedKey]"
```

Comment:
update apiVersion or just remove it

Reply:
Updated, not sure of the syntax of removing